test/e2e/crs/wiremock-config.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: wiremock-config
+spec:
+  restartPolicy: Never
+  containers:
+  - name: curl
+    image: curlimages/curl:latest
```

Comment:
Creative solution. We should consider adding config map support to the Agent CRD...